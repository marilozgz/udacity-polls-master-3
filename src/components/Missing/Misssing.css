h1.content-title {
    font-weight: bold;
    font-size: 25px;
  }
  
  .content {
    background-image: url(../../images/Contentbg.jpeg);
    margin-top: 14%;

  }
  
  .content-button {
    background-color: #f9ca24;
    font-size: 14px;
    font-weight: bold;
  }