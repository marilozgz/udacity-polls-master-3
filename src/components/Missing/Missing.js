import { Link } from "react-router-dom"
import {  useNavigate } from "react-router-dom";

import React from 'react';
import Moon  from '../../images/moon.png';
import Rocket from '../../images/rocket.png';
import "./Misssing.css"
const Missing = () => {

    return (

        <div className="content row">
            <div className="content-col-left col-md p-5 align-self-center">
                <img src={ Moon } alt="" className="img-fluid"/>
            </div>
            <div className="content-col-right col-md p-5 align-self-center text-light">
                <h1 className="content-title text-left">404 Not Found</h1>
                <p className="text-left">We sorry, the page you are looking for does not appear. Please back home by pressing button bellow.</p>
              
              
                <img src={ Rocket } alt="" className="img-fluid"/>
            </div>
        </div>
    )
}

export default Missing