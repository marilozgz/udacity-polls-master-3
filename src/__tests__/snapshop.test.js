import { render } from "@testing-library/react";
import React from "react";
import Missing from "../components/Missing/Missing";

describe("Missing", () => {
  it("will match snapshot", () => {
    const { container } = render(<Missing />);
    expect(container).toMatchSnapshot();
  });
});