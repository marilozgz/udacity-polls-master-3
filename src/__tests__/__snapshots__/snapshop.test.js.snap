// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Missing will match snapshot 1`] = `
<div>
  <div
    class="content row"
  >
    <div
      class="content-col-left col-md p-5 align-self-center"
    >
      <img
        alt=""
        class="img-fluid"
        src="moon.png"
      />
    </div>
    <div
      class="content-col-right col-md p-5 align-self-center text-light"
    >
      <h1
        class="content-title text-left"
      >
        404 Not Found
      </h1>
      <p
        class="text-left"
      >
        We sorry, the page you are looking for does not appear. Please back home by pressing button bellow.
      </p>
      <img
        alt=""
        class="img-fluid"
        src="rocket.png"
      />
    </div>
  </div>
</div>
`;
